config/checkstyle/checkstyle.xml

```diff
@@ -26,7 +26,12 @@
   <module name="FileTabCharacter">
     <property name="eachLine" value="true"/>
   </module>
-
+  <module name="RegexpMultiline">
+    <property name="format" value="^([^\n ]+ )*(class|interface) [^{]*\{\n[^\n}]"/>
+    <property name="message" value="Leave empty line after class/interface definition!"/>
```

Comment:
hmm I don't personally agree with this. Thoughts?

Reply:
I have added this after few review comments to add new line in my recent PR :D

Reply:
I have been saying to remove lines 😂 Sounds like we need to catch up as team

Reply:
As discussed offline, let's leave it there to have a consistent code base.